tests/sql/moz-fx-data-shared-prod/fenix_derived/firefox_android_clients_v1/test_first_session_no_baseline/expect.yaml

```diff
@@ -0,0 +1,40 @@
+---
```

Comment:
client-2 reported baseline ping.
client-1 and client-3 didn't report a baseline ping but still do appear with their corresponding values for reported metrics and first_session ping.